Сетка страниц и крупных блоков на флексах

ValentinaKobtseva opened the pull request:
---
:mortar_board: [Сетка страниц и крупных блоков на флексах](https://up.htmlacademy.ru/univer-html1/2/user/2579379/tasks/9)

:boom: https://htmlacademy-univer-htmlcss-1.github.io/2579379-sedona-2/5/

ValentinaKobtseva committed:
Сетка страниц и крупных блоков на флексах

OAPrilepa commented on index.html:
```diff
@@ -125,39 +125,44 @@ <h2 class="subscription-heading subscription-heading-bright">Подпишите
+          </addres>
+        </section>
+        <section class="htmlacademy">
           <a class="htmlacademy-link" href="https://htmlacademy.ru/">.html academy</a>
```
Давай добавим здесь target=_blank - чтобы ссылка на внешний (для нашей странички) ресурс открывалась по-умолчанию в новой вкладке.

ValentinaKobtseva replied:
Добавила, полезный атрибут, спасибо)

OAPrilepa commented on index.html:
```diff
@@ -125,39 +125,44 @@ <h2 class="subscription-heading subscription-heading-bright">Подпишите
+        <h2 class="visually-hidden">Социальные сети</h2>
+        <ul class="footer-sotial-list list">
+          <li class="footer-social-item">
+            <a class="button" href="https://vk.com/htmlacademy">
```
И на ссылках в соц. сети - тоже target=_blank

OAPrilepa commented on catalog.html:
```diff
@@ -230,7 +231,7 @@ <h3 class="visually-hidden">Информайия о рейтинге гости
         </section>
 
-        <section class="subscribtion-section">
+        <section class="subscribtion-section catalog-subscribtion-section">
```
Давай еще здесь добавим на поле ввода required аттрибут, чтобы браузер не давал отправить форму без заполнения поля, в сочетании с type=email он будет требовать ввода корректного адреса.

ValentinaKobtseva replied:
добавила

OAPrilepa commented on index.html:
```diff
@@ -58,15 +58,15 @@ <h3 class="advantages-header-item advantages-header-bright">Настоящий 
             <p class="advantages-description description-bright">Седона — не аттракцион для туристов, там течёт своя жизнь</p>
```
Вот тут все слиплось, так не задумано:

<img width="448" alt="image" src="https://github.com/user-attachments/assets/39036f85-f344-49b7-9d7b-63be0653d537">

ValentinaKobtseva replied:
Разве этот и ниже - не мелкие блоки, верстка которых предполагается в 6 задании?

OAPrilepa replied:
Можно и отложить, конечно, смотри как удобней. Просто сразу бросается в глаза, в других работах такого не видел)

ValentinaKobtseva replied:
Ты прав, бросается) Исправила. Ну и в остальных преимуществах тоже вдогонку

OAPrilepa commented on index.html:
```diff
@@ -58,15 +58,15 @@ <h3 class="advantages-header-item advantages-header-bright">Настоящий 
+          <li class="advantages-item advantages-item-light advantages-item-without-image">
             <h3 class="advantages-header-item">Мало туристов</h3>
             <p class="advantages-description">Большинство едет в Гранд Каньон и толпится там</p>
           </li>
```
И тут тоже:

<img width="451" alt="image" src="https://github.com/user-attachments/assets/6f77823d-4498-42f7-b9de-d0319955d222">

OAPrilepa commented on index.html:
```diff
@@ -125,39 +125,44 @@ <h2 class="subscription-heading subscription-heading-bright">Подпишите
     </main>
+  </div>
 
     <footer class="footer-main">
```
У подвала потерялась обертка и на ширине выше 1 200 нет полей, как у шапки и основного контента страницы:

<img width="239" alt="image" src="https://github.com/user-attachments/assets/f214035e-66ca-4e98-b57e-d3950aad8c2d">

ValentinaKobtseva replied:
Исправила

ValentinaKobtseva committed:
Правки

Добавила атрибуты к инпутам, отцентрировала текст в преимуществах, поправила footer
